src/Monitor/Agents.vue

```diff
@@ -57,8 +57,8 @@ export default {
       }
     },
     query: async function() {
-      this.groups = await this.$agent.p_mfa('ws_db_agent_group', 'get')
-      this.agents = await this.$agent.p_mfa('ws_admin', 'agents', ['all'])
+      this.groups = await this.$agent.p_mfa('ws_agent_agent_group', 'get')
```

Comment:
Should be ws_agent, agent_groups

Reply:
p_mfa('ws_agent', 'agent_group', 'get') ?

Reply:
p_mfa('ws_agent', 'agent_groups')